sdk/samples/iot/paho_iot_hub_twin_sample.c

```diff
@@ -329,7 +334,7 @@ static void send_reported_property(void)
            twin_patch_topic_buffer,
            az_span_size(reported_property_payload),
            az_span_ptr(reported_property_payload),
-           0,
+           IOT_SAMPLE_MQTT_PUBLISH_QOS,
```

Comment:
Can we use the public enum that we expose to the users rather than a sample one?
`AZ_HUB_CLIENT_DEFAULT_MQTT_TELEMETRY_QOS`

Reply:
I had considered it, but the issue is that QOS value is used to publish messages that have nothing to do with telemetry, so I think it would mislead the customer. (e.g., method response)